test/configs/testing-rpmlintrc

```diff
@@ -33,7 +33,6 @@ addFilter("doc-file-dependency .*")
 addFilter("doc-file-dependency .*\.py ")addFilter("doc-file-dependency .*\.py ")addFilter("E: devel-file-in-non-devel-package .*site-packages.*fake_libc_include.*")
 addFilter("E: devel-file-in-non-devel-package")
 addFilter("E: devel-file-in-non-devel-package .*site-packages.*fake_libc_include.*")
-addFilter("E: shlib-policy-name-error")
```

Comment:
This is just a list of random values not relevant to reality, why remove from here?

Reply:
OK, I have removed the second commit